meeteval/io/ctm.py

```diff
@@ -131,6 +131,23 @@ def grouped_by_filename(self) -> Dict[str, 'CTMGroup']:
     def grouped_by_speaker_id(self) -> Dict[str, CTM]:
         return self.ctms
 
+    def to_stm(self):
+        from meeteval.io import STM, STMLine
+        stm = []
+        for filename, v1 in self.grouped_by_filename().items():
+            for speaker_id, v2 in v1.grouped_by_speaker_id().items():
+                for line in v2.lines:
+                    stm.append(STMLine(
+                        filename=filename,
+                        channel=0,
```

Comment:
We should check, what the default for channel usually is.
Often I did it wrong. I think I also used 0, while it should be 1.

Reply:
The "official" documentation (https://github.com/usnistgov/SCTK/blob/master/doc/infmts.htm) sais:

> The waveform channel.   The text of the waveform channel is not restricted by sclite.  The text can be any text string without witespace so long as the matching string is found in both the reference and hypothesis input files.

This is the only documentation I know about the STM format, so there is no official default defined. Where did you take the 1 from?

Reply:
Took some time to find it.
 - On https://github.com/nryant/dscore I found `channel (1-indexed) that turn is on; should always be 1`. (Currently, the best argument to select a default, because it is the only argument)
 - From the RWTH I saw an STM file and there was the default 1.
 - In ESPnet I saw some rttm files and they have as default 1.

While that is not a strong argument, it is at least an argument to select one over another.